pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -371,3 +374,31 @@ def log_train_step_metrics(self, batch_output):
             if len(batch_log_metrics) > 0 or len(grad_norm_dic) > 0:
                 self.log_metrics(batch_log_metrics, grad_norm_dic)
                 self._callback_metrics.update(batch_log_metrics)
+
+    @property
+    def evaluation_log_step(self) -> Optional[int]:
```

Comment:
why optional?

Reply:
Just because running stage can be something other than validating or testing, so it's possible for the method to return None (though not the way we use it)

Reply:
could we make it more clear by adding
```
else:
    return None
```
at the end?